_layouts/footerimage.html

```diff
@@ -0,0 +1,40 @@
+<!doctype html>
+<html lang="en" xmlns:fb="http://ogp.me/ns/fb#">
+
+    {% include head.html %}
+
+    <body id="top">
+        <div class="container">
+            {% include nav.html %}
+        </div>
+
+        <div class="container">
+            {{ content }}
+        </div>
+
+        <div class="footer-image" style="background-image: url('{{ site.baseurl }}/static/img/footers/{{ page.footerimage }}')">
```

Comment:
A separate layout for the single addition is unnecessary.

Add this to the beginning of **_includes/footer.html**:

```
{% if page.footer_image %}
    ...<div ...>...
{% endif %}
```

don't forget to adjust `{{ page.footerimage }}` to say `{{ page.footer_image }}`